agent/tool-scripts/tests/blktrace/gold/stderr

```diff
@@ -2,12 +2,20 @@ blktrace: Error, one of the following options is required, --install|--start|--s
 
 The following options are available:
 
+Operational options - do *not* specify any of these when you register a tool:
+
 	--install                 verify this perf tool will work
 
 	--start|stop|postprocess  start/stop/postprocess the data collection
 	-d str|--dir=str          directory to store data collection (required)
+
+Tool-specific options - these should be specified when you register a tool:
+
 	--devices=str,[str]       the list of block devices to trace (required w/ --start)
```

Comment:
I really don't want to open this can of worms, but....

The text at line 12 says I should specify the option described at line 14, which in turn suggests (at least on a light read) that I need to provide `--start`, which the text at line 5 tells me not to use...which feels like a contradiction (assuming that I as a reader managed to acquire all of these details...).

Now, the text at line 14 is _actually_ saying that `--devices` needs to be specified **when** `--start` is specified, but the typical user may not have the perspective to know that the Agent is going to run this command for him/her and that when it does it will provide `--start` and, most importantly, it will provide `--devices` using the information from the registration (and, thus, the user _should_ provide `--devices` and should _not_ provide `--start`...).

Now, how we capture all that in 60 or so characters is beyond me at the moment...but, perhaps the first step is to just remove the `(required w/ --start)` text?

Ditto for `bpftrace`.

Reply:
So `--devices` is required when registering `blktrace` (and similarly `--script` is required when registering `bpftrace`) as far as the user is concerned. I just deleted the qualification.